install/oddjob/org.freeipa.server.trust-enable-agent

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/libexec/ipa/ipa-trust-enable-agent "$1" 2>&1
```

Comment:
What about moving the whole helper into this place instead of using, effectively, a double wrapper?